mongo/change_stream.go

```diff
@@ -287,7 +287,7 @@ func (cs *ChangeStream) executeOperation(ctx context.Context, resuming bool) err
 	// Execute the aggregate, retrying on retryable errors once (1) if retryable reads are enabled and
 	// infinitely (-1) if context is a Timeout context.
 	var retries int
-	if cs.client.retryReads && cs.wireVersion != nil && cs.wireVersion.Max >= 6 {
+	if cs.client.retryReads && cs.wireVersion != nil {
```

Comment:
```suggestion
	if cs.client.retryReads {
```
The previous check seemed to set `retries = 1` if retryable reads were enabled and the wire version was known and greater than or equal to 6. I believe the `cs.wireVersion != nil` check here used the < 6 wire version logic when the wire version was unknown. I think we can remove it and always operate under the assumption that wire version is greater than 6 given that we no longer support earlier wire versions.